capstone-client/src/app/routing/app-routing.module.ts

```diff
@@ -4,12 +4,14 @@ import {SignInComponent} from '../sign-in/sign-in.component';
 import {SignUpClientComponent} from '../sign-up/sign-up-client.component';
 import {SignUpSupplierComponent} from '../sign-up/sign-up-supplier.component';
 import {HomeComponent} from '../home/home.component';
+import {ForgetComponent} from '../forget/forget.component';
```

Comment:
```suggestion
import {ForgetComponent} from '../forget/forget.component';
```

same as above add these things as a new request